packages/devtools_shared/lib/src/extensions/extension_manager.dart

```diff
@@ -78,6 +78,8 @@ class ExtensionsManager {
       }
       for (final extension in extensions) {
         final config = extension.config;
+        final isPublic =
+            extension.rootUri.path.contains(path.join('.pub-cache', 'hosted'));
```

Comment:
add a TODO and file a bug to not depend on the internals of how the pub-cache directories are specified.